### DISCUSSION TITLE
Merge multiple calendars of different users into a single, global one

### DISCUSSION THREAD

**Discussion by NicoHood** [Kozea/Radicale#1601] [Q&A general]
My usecase:

I have multiple users with a calendar each. They all use it to report their events to me. For the final audience I'd like to have a merge of all those different calendars as caldav. This way nobody can manipulate the event of each other, making it harder for trolls and I can easily detect who is publishing fake events. I dont expect that to happen, but you never know!

Is there a way to do that with radicale currently?

**Comment by Unrud** [comment]
You could use [storage-hook](https://radicale.org/3.0.html#documentation/configuration/storage/hook) to run a script that copies all `*.ics` from the user calendars directories into a new calendar directory.
